meteor/client/ui/SegmentTimeline/Parts/OutputGroup.tsx

```diff
@@ -42,6 +42,7 @@ interface IOutputGroupProps {
 	onContextMenu?: (contextMenuContext: IContextMenuContext) => void
 	indexOffset: number
 	isPreview: boolean
+	showDurationSourceLayers?: Set<string>
```

Comment:
What does `showDurationSourceLayers` do? Why is it a `Set<string>`?

Reply:
It's a set of the sourcelayers for which the duration of the piece should be shown (e.g. only VTs and VOs should have timing shown on the Piece. I don't know why it's a set instead of an array.

Reply:
Documentation is on the `RundownViewLayout` interface https://github.com/nrkno/sofie-core/blob/0633774ad3c69186194a9bc7dd5d2f062143e6f9/meteor/lib/collections/RundownLayouts.ts#L367

Reply:
Yeah, but why Set of `string` as opposed to Set of `ISourceLayer['_id']`? String is very opaque.

Reply:
Using `ISourceLayer['_id']` doesn't seem to do anything in terms of type checking. It still resolves to `Set<string>` and I still can do silly things like `showDurationSourceLayers.has(sourceLayer.name)`, but you want it to just be a hint to the developers?